test/journeys/specs/recents/dataApi/basic.js

```diff
@@ -49,7 +56,9 @@ describe(`Widget Recents`, () => {
             }
           }
         });
-        return marty.spark.internal.mercury.connect();
+        return marty.spark.internal.device.register()
+        .then(() => marty.spark.internal.feature.setFeature(`developer`, FEATURE_FLAG_GROUP_CALLING, true))
```

Comment:
Nitpick, since we don't have the eslint rule yet. Indent here and next line unless it's off because of the diff.

Reply:
Fixed.